addons/test_mail_full/tests/test_portal.py

```diff
@@ -295,45 +279,53 @@ def test_customer_access_logged_without_access(self):
         self.authenticate(self.env.user.login, self.env.user.login)
         res = self.url_open(self.record_access_url)
         self.assertEqual(res.status_code, 200)
-        self.assert_URL(res.url, f'/my/test_portal/{self.record_portal.id}')
+        parsed_url = url_parse(res.url)
+        self.assertEqual(parsed_url.path, f'/my/test_portal/{self.record_portal.id}')
 
     @users('portal_test')
     def test_customer_access_logged_without_access_wrong_token(self):
         """Check that it redirects to discuss when logged customer has no access to the record and token is invalid."""
         self.authenticate(self.env.user.login, self.env.user.login)
         res = self.url_open(self.record_access_url_wrong_token)
         self.assertEqual(res.status_code, 200)
-        self.assert_URL(res.url, '/my', {'action': 'mail.action_discuss'})
+        parsed_url = url_parse(res.url)
+        self.assertEqual(parsed_url.path, '/my')
+        self.assertEqual(parsed_url.query, 'subpath=action-mail.action_discuss')
```

Comment:
Idem as previous comment

Reply:
Previously we were getting value of `res.url` in above test case as -
`http://127.0.0.1:8069/my#action=mail.action_discuss`

But after the conversion of urls in `/community/addons/mail/controllers/mail.py` we are getting value of `res.url` in above test case as -
`http://127.0.0.1:8069/my?subpath=action-mail.action_discuss`

You can refer to [Runbot Build](https://runbot.odoo.com/runbot/build/65968276) for more reference.
That's why I need to modify the test case. Thanks!!